WordPress/Classes/ViewRelated/Aztec/Stock Photos/StockPhotosDataSource.swift

```diff
@@ -11,6 +11,7 @@ final class StockPhotosDataSource: NSObject, WPMediaCollectionDataSource {
 
     private let throttle = Throttle(seconds: 1)
 
+    var searchQuery: String = ""
```

Comment:
Would it make sense to make this var private? I don't see this as a blocker anyway